PoCs/README.md

```diff
@@ -0,0 +1,139 @@
+# PoCs
+
+This folder is dedicated to hosting various PoCs for testing and demonstrating
+potential configurations or enhancements in the Metal3 environment.
+
+## PoCs Overview
+
+### 1\. PoC Hostnetworkless Ironic with VirtualMedia Using NodePorts
+
+This PoC is part of the discussion [metal3-io/baremetal-operator#\
+1739](https://github.com/metal3-io/baremetal-operator/discussions/1739) and\
+demonstrates running Ironic without `hostNetwork`, limited to the VirtualMedia\
+use case, using a NodePort service.
+
+#### **Branch:**
+
+Run this PoC or review code changes at [hostnetworkless-ironic-\
+nodeport](https://github.com/Nordix/metal3-dev-env/tree/test-svc-nodePort/mboukhalfa)
+
+#### Key Changes
+
+1. **Edit Ironic Deployment:**
+
+- Remove `hostNetwork: true`.
```

Comment:
This needs to intended, otherwise they reset the numbered list, and all items are `1.`

Reply:
`PoCs/README.md:24:1 MD007/ul-indent Unordered list indentation [Expected: 0; Actual: 1]` not sure why it expect 0

Reply:
It should expect 3 per indentation level. Maybe it thinks 1 is just mistake.